src/Device.js

```
@@ -70,21 +70,17 @@ export default class Device extends EventEmitter {
   * @param {boolean} autoStart if set to true the device enables default sensors and starts measurements.
   */
   async open(autoStart = false) {
-    try {
-      await this._connect();
-      await this._init();
-      await this._getStatus();
-      await this._getDeviceInfo();
-      await this._getDefaultSensorsMask();
-      await this._getAvailableSensors();
-
-      this._onOpened();
-
-      if (autoStart) {
-        this.start();
-      }
-    } catch (err) {
-      console.error(err);
```

It seems like we should throw a new error here, rather than remove our catch